actors/builtin/miner/miner_test.go

```diff
@@ -1703,6 +1706,67 @@ func TestTerminateSectors(t *testing.T) {
 			assert.Equal(t, big.Zero(), st.InitialPledgeRequirement)
 		}
 	})
+
+	t.Run("charges correct fee for young termination of committed capacity upgrade", func(t *testing.T) {
+		actor := newHarness(t, periodOffset)
+		rt := builderForHarness(actor).
+			WithBalance(bigBalance, big.Zero()).
+			Build(t)
+		actor.constructAndVerify(rt)
+
+		// A miner will pay the minimum of termination fee and locked funds. Add some locked funds to ensure
```

Comment:
Because you are setting your balance to `bigBalance` this doesn't seem true.  It will deduct from locked funds first and then from the unlocked balance.